chain/client/src/metrics.rs

```diff
@@ -135,6 +135,26 @@ pub(crate) static VALIDATORS_CHUNKS_EXPECTED_BY_SHARD: Lazy<IntGaugeVec> = Lazy:
     .unwrap()
 });
 
+pub(crate) static VALIDATORS_CHUNK_ENDORSEMENTS_PRODUCED_BY_SHARD: Lazy<IntGaugeVec> =
+    Lazy::new(|| {
+        try_create_int_gauge_vec(
+            "near_validators_chunk_endorsements_produced_by_shard",
+            "Number of chunk endorsements produced by a chunk validator",
+            &["account_id", "shard_id"],
```

Comment:
I heard this is very expensive in grafana because you effectively create a new metric for every (validator, shard_id) pair. That being said it seems to be the pattern here so 😇

Reply:
I would remove the non-shard metrics, I think shard-based metrics will naturally be needed in the future until the shards become balanced.